### PR TITLE
Change the config field name `root` to `ca_certs`

### PR DESCRIPTION
**Pull request by div-seungha**
Fixes: #91 

This is my first time working in this repository, so I had some difficulty understanding the context. I referred to the work that has already been merged in other repositories while making my changes, but there may be mistakes or errors in my modifications. If you notice any problems, I would appreciate it if you could point them out. I will correct them promptly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Example
 ```toml
 key = "key.pem"
 cert = "cert.pem"
-root = "root.pem"
+ca_certs = "ca_certs.pem"
```
Comment (sophie-cluml) [inline README.md:43]:
```
ca_certs = ["ca_cert1.pem", "ca_cert2.pem"]
```

이렇게 되는 것이 도달해야할 스테이트입니다.

Comment (div-seungha) [reply README.md]: `ca_certs` 값이 여러 경로를 받을 수 있도록 관련 로직들을 수정해 보았는데, 리뷰 부탁드립니다.

**Comment by kimhanbeom** [comment]
https://github.com/aicers/crusher/blob/c3f0eafaec13e6b271b7aef0dbea855f18aa6517/src/settings.rs#L27-L29

- In order for multiple paths to be read from the config file, it must be modified as follows: `PathBuf` -> `Vec<PathBuf>`

File: tests/config.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 key = "tests/key.pem"
 cert = "tests/cert.pem"
-root = "tests/root.pem"
+ca_certs = "tests/ca_certs.pem"
```
Comment (kimhanbeom) [inline tests/config.toml:3]:
```suggestion
ca_certs = ["tests/ca_certs.pem"]
```

**Review by kimhanbeom — CHANGES_REQUESTED** [review]

File: tests/config.toml (hunk 2 above)
Comment (div-seungha) [reply tests/config.toml]: I modified the code as you showed me.

**Review by kimhanbeom — APPROVED** [review]

File: tests/config.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 key = "tests/key.pem"
 cert = "tests/cert.pem"
-root = "tests/root.pem"
+ca_certs = ["tests/ca_certs.pem"]
```
Comment (sophie-cluml) [inline tests/config.toml:3]:
```suggestion
ca_certs = ["tests/ca_cert.pem"]
```

I'd like to suggest this small change. I think the file also needs renaming to `ca_cert.pem`.

File: CHANGELOG.md [hunk 4]
```diff
@@ -14,6 +14,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Changed
 
+- Renamed the root field to ca_certs and updated it to support multiple CA
+  certificates.
```
Comment (sophie-cluml) [inline CHANGELOG.md:18]: I think this line needs to be combined with this line https://github.com/aicers/crusher/pull/94/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR33
```
- Changed configuration fields name.
  - `roots` to `root`.
```

So I would like to suggest removing L17-18, and instead modifying L33 to

``` 
  - `roots` to `ca_certs`. It also introduces support for multiple CA certificates.
  certificates.
```

File: src/subscribe/tests.rs [hunk 5]
```diff
@@ -16,14 +16,14 @@ use super::{Client, Conn};
...
-    to_cert_chain, to_private_key, to_root_cert,
+    to_ca_certs, to_cert_chain, to_private_key,
 };
 
 pub(crate) static TOKEN: LazyLock<Mutex<u32>> = LazyLock::new(|| Mutex::new(0));
 
 const CERT_PATH: &str = "tests/cert.pem";
 const KEY_PATH: &str = "tests/key.pem";
-const CA_CERT_PATH: &str = "tests/root.pem";
+const CA_CERT_PATH: &str = "tests/ca_certs.pem";
```
Comment (sophie-cluml) [inline src/subscribe/tests.rs:26]:
```suggestion
const CA_CERT_PATH: &str = "tests/ca_cert.pem";
```

This is a suggestion in line with the previous comment.

**Review by sophie-cluml — COMMENTED** [review]
👍  I think it looks good, and I left a few comments that I would appreciate if you could take into consideration.

File: CHANGELOG.md [hunk 6]
```diff
@@ -28,11 +28,12 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   - If the reload trigger succeeds, the new configuration is applied from the
     temporary file; otherwise, the temporary file is deleted.
 - Changed configuration fields name.
-  - `roots` to `root`.
+  - `roots` to `ca_certs`. It also introduces support for multiple CA
+     certificates. certificates.
```
Comment (sophie-cluml) [inline CHANGELOG.md:32]: certificates. certificates.

두번 들어간것 같아요. 제가 잘못 두 번 썼었네요.

Comment (div-seungha) [reply CHANGELOG.md]: 앗 저도 확인을 꼼꼼히 했어야 했는데 죄송합니다

**Review by sophie-cluml — APPROVED** [review]
LGTM

File: tests/config.toml (hunk 3 above)
Comment (div-seungha) [reply tests/config.toml]: As you pointed out, I modified the path name.

File: CHANGELOG.md (hunk 4 above)
Comment (div-seungha) [reply CHANGELOG.md]: I removed the first line and combined with that line as you pointed out.

File: src/subscribe/tests.rs (hunk 5 above)
Comment (div-seungha) [reply src/subscribe/tests.rs]: I modified the path name.

**Pull request merged**